src/Microsoft.OData.Core/UriParser/Binders/InBinder.cs

```diff
@@ -331,10 +352,22 @@ private static string NormalizeGuidCollectionItems(string bracketLiteralText)
         private static string NormalizeDateTimeCollectionItems(string bracketLiteralText)
         {
             string normalizedText = bracketLiteralText.Substring(1, bracketLiteralText.Length - 2).Trim();
+
+            // If we have empty brackets ()
             if (normalizedText.Length == 0)
             {
                 return "[]";
             }
+
+            string normalizedTextWithoutSpaces = normalizedText.Replace(" ", string.Empty);
+
+            // If we have '' or "" or empty strings in the brackets e.g (' ')
+            // We throw an exception since empty strings are not valid guid values.

```

Comment:
Just a type. This method normalized collections of DateTime, not Guid.

Reply:
Fixed